### DISCUSSION TITLE
Are there more seconds in a week ...

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#769] [Galaxy Trivia] [answered]
... or feet in 100 miles?

**Bonus:** Pin 📌 on a map of the place whose zip code matches the difference.

**Comment by peterbaccaro** [comment, marked as answer]
There are **more seconds in a week** than there are feet in 100 miles.

To calculate the number of seconds in a week, we can use the fact that there are 60 seconds in a minute, 60 minutes in an hour, 24 hours in a day, and 7 days in a week:

60 seconds/minute * 60 minutes/hour * 24 hours/day * 7 days/week = 604,800 seconds/week

To calculate the number of feet in 100 miles, we can use the fact that there are 5,280 feet in a mile:

100 miles * 5,280 feet/mile = 528,000 feet

The difference is 604,800 - 528,000 = 76800 and this as a zip code on a map is as follows: 

![image](https://github.com/ParanoidUser/yolo/assets/74666079/93728455-8cb7-4c6d-8d9e-fa4b128524d1)
